### ISSUE TITLE
DR 18 splash page

### ISSUE THREAD

**Issue by Sean-Morrison** [sdss/flipper#3]
The screenshots for the various pages on the DR18 splash page (https://dr18.sdss.org/) is showing DR14 versions of the pages. Some of these (SkyServer, SDSS Home) have significant updates, but all should be updated.

**Comment by havok2063** [comment]
I can see about updating them, but I'm not sure we really need to keep these updated for each data release as the content can change quite frequently.  They're more like representative screenshots.  But maybe once a phase is a good cadence.

**Comment by Sean-Morrison** [comment]
I agree that we do not need to do it often, but I figured it was time since we are now have SDSS-V data and sites public and the screenshots have SDSS-IV logos.
